src/clj/puppetlabs/services/request_handler/request_handler_core.clj

```diff
@@ -259,73 +251,13 @@
   "Make the request mutable.  This is required by the ruby layer."
   (HashMap. request))
 
-(defn bad-request?
```

Comment:
All of the deleted functions here moved over to the new `jruby-request` namespace.  Other changes in this namespace are mostly just about referencing the functions from their new location.